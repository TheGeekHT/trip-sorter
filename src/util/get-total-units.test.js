import getTotalUnits from './get-total-units'
import trips from 'deals'

const tripsToCheck = [
  trips[0],
  trips[1],
  trips[2],
];

describe('get total units helper', () => {
  test('get proper total values', () => {
    const result = getTotalUnits(tripsToCheck);
    expect(result).toMatchSnapshot();
  });
});