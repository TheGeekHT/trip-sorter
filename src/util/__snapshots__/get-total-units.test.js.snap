exports[`get total units helper get proper total values 1`] = `
Object {
  "cost": 310,
  "stops": 2,
  "time": Object {
    "h": 17,
    "m": 30,
  },
  "transport": Array [
    "train",
    "bus",
    "car",
  ],
}
`;
